buildConfig/build-extras.gradle

```diff
@@ -158,6 +159,7 @@ android {
             resValue("string", "deeplink_base_url", "${staging_deeplink_base_url}")
             resValue("string", "custom_scheme_url", "${staging_custom_scheme}")
             buildConfigField 'String', 'BASE_URL', "\"${staging_base_url}\""
+            buildConfigField 'String', 'MERGE_ACCOUNT_BASE_URL', "\"${staging_base_url}\""
```

Comment:
It should have new variable for all variants like "staging_merge_base-url".

Reply:
Resolved: added keys - production_merge_account_base_url, staging_merge_account_base_url, dev_merge_account_base_url